lib/capybara/screenshot/diff/test_methods.rb

```diff
@@ -88,7 +100,9 @@ def assert_image_not_changed(caller, name, comparison)
 
           return unless result
 
-          "Screenshot does not match for '#{name}' #{comparison.error_message}\nat #{caller.join("\n")}"
+          result = +"Screenshot does not match for '#{name}' #{comparison.error_message}"
+          result += "\nat #{caller.join("\n")}" if backtrace
```

Comment:
@UweKubosch @donv, what do you think I'll remove to show the backtrace in the message and test the framework to show the backtrace instead?

Reply:
Yeah, if the test framework can show the stack trace, that would be better.